includes/admin/class-support-admin.php

```diff
@@ -18,17 +18,6 @@ class NerdPress_Admin {
 	public function __construct() {
 		add_action( 'admin_menu', array( $this, 'settings_menu' ), 59 );
 		add_action( 'admin_init', array( $this, 'settings_tabs' ) );
-		add_action( 'admin_head', array( $this, 'hide_wp_rocket_beacon' ) );
-	}
-
-	/**
-	 * Hide WP Rocket's help beacon.
-	 */
-	public function hide_wp_rocket_beacon() {
```

Comment:
@CNoemi94 I think we do want to keep this. Since it was there to not allow clients to use the WP Rocket beacon. It's our account and we don't want them contacting WP Rocket directly.